pom.xml

```diff
@@ -3,15 +3,25 @@
          xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.hibernate</groupId>
+    <groupId>org.hibernate.lucene-modules</groupId>
     <artifactId>lucene-modules-parent</artifactId>
     <name>Lucene JBoss Modules parent</name>
     <packaging>pom</packaging>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>5.5.0-SNAPSHOT</version>
 
     <properties>
-        <version.lucene>4.9.0</version.lucene>
-        <version.wildfly>8.1.0.Final</version.wildfly>
+        <version.lucene>5.5.4</version.lucene>
+        <!-- This should match the major.minor of the Lucene version, optionally revisioned with module micro fixes -->
+        <slot.simple.id>5.5</slot.simple.id>
+        <version.wildfly>10.1.0.Final</version.wildfly>
+        <!-- Integration test dependencies; typically need to match WildFly version -->
+        <arquillianVersion>1.1.11.Final</arquillianVersion>
```

Comment:
I'm being fussy but... Is there a particular reason to use multiple naming schemes for version properties? So far I've seen `lucene.version` (in a test file), `version.lucene` (just above) and `arquillianVersion` (this line). Maybe we should make up our mind... ?